deployments/base/usdbc/migrations/1689892563_configurate_and_ens.ts

```diff
@@ -0,0 +1,240 @@
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(actions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async verify(deploymentManager: DeploymentManager, govDeploymentManager: DeploymentManager, preMigrationBlockNumber: number) {
+    const ethers = deploymentManager.hre.ethers;
+    await deploymentManager.spider(); // We spider here to pull in Base COMP now that reward config has been set
+
+    const {
+      comet,
+      rewards,
+      COMP,
+      USDbC
+    } = await deploymentManager.getContracts();
+
+    const {
+      comptrollerV2,
+    } = await govDeploymentManager.getContracts();
+
+    // 1.
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    // TODO: uncomment when contracts are deployed
```

Comment:
just a reminder to uncomment out this test before merging

Reply:
Uncommented